crates/uv/src/commands/python/uninstall.rs

```diff
@@ -83,7 +84,7 @@ pub(crate) async fn uninstall(
         return Ok(ExitStatus::Failure);
     }
 
-    let tasks = futures::stream::iter(matching_installations.iter())
+    let tasks = futures::stream::iter(matching_installations.iter().unique())
```

Comment:
Can we collect into a BTreeSet beforehand or no?

Reply:
Hm `matching_installations` is  `BTreeSet` already for deterministic ordering.

Reply:
Ah, I did not realize that, I've raised #4842 to revert this.